store/src/test/java/com/zimbra/cs/service/admin/IssueCertTest.java

```diff
@@ -108,37 +109,55 @@ public void clearData() {
 
   @AfterClass
   public static void tearDown() {
-    mockedStatic.close();
+    staticRemoteManager.close();
+    staticRemoteCertbot.close();
   }
 
   @Test
-  public void shouldReturnSuccessMessageIfAllChecksPassed() throws Exception {
-    createInfrastructure();
+  public void shouldSupplyAsyncAndReturnResponse() throws Exception {
+    domainAttributes.put(ZAttrProvisioning.A_zimbraPublicServiceHostname, publicServiceHostName);
+    domainAttributes.put(ZAttrProvisioning.A_zimbraVirtualHostname, virtualHostName);
 
-    final String result = "SUCCESS";
+    Domain expectedDomain = provisioning.createDomain(domainName, domainAttributes);
 
-    when(remoteManager.execute(command)).thenReturn(remoteResult);
-    when(remoteResult.getMStdout()).thenReturn(result.getBytes());
+    final String serverName = "serverName";
+    final Server server =
+        provisioning.createServer(
+            serverName,
+            new HashMap<>() {
+              {
+                put(ZAttrProvisioning.A_cn, serverName);
+                put(ZAttrProvisioning.A_zimbraServiceEnabled, Provisioning.SERVICE_PROXY);
+              }
+            });
 
-    final Element response = handler.handle(request, context);
-    final Element message = response.getElement(E_MESSAGE);
-    assertEquals(message.getAttribute(A_DOMAIN), domainName);
-    assertEquals(message.getText(), result);
-  }
+    staticRemoteManager.when(() -> RemoteManager.getRemoteManager(server))
+        .thenReturn(remoteManager);
+    staticRemoteCertbot.when(() -> RemoteCertbot.getRemoteCertbot(remoteManager))
+        .thenReturn(remoteCertbot);
 
-  @Test
-  public void shouldReturnFailureMessageIfRemoteExecutionFailed() throws Exception {
-    createInfrastructure();
+    Mailbox expectMailbox = getRequestedMailbox(zsc);
 
-    final String result = "FAILURE";
-    final String expectedMessage = "system failure: FAILURE";
+    String expectedCommand = "certbot certonly --agree-tos --email admin@example.com"
+        + " -n --keep --webroot -w /opt/zextras "
+        + "--cert-name example.com "
+        + "-d public.example.com -d virtual.example.com";
 
-    when(remoteManager.execute(command)).thenThrow(ServiceException.FAILURE(result));
+    when(remoteCertbot.createCommand(
+        RemoteCommands.CERTBOT_CERTONLY,
+        mail,
+        AdminConstants.DEFAULT_CHAIN,
+        domainName,
+        publicServiceHostName,
+        expectedDomain.getVirtualHostname())).thenReturn(expectedCommand);
 
     final Element response = handler.handle(request, context);
     final Element message = response.getElement(E_MESSAGE);
+
     assertEquals(message.getAttribute(A_DOMAIN), domainName);
-    assertEquals(message.getText(), expectedMessage);
+    assertEquals(message.getText(), IssueCert.RESPONSE);
```

Comment:
```suggestion
    assertEquals(IssueCert.RESPONSE, message.getText());
```
correctly order expected and actual values.